tests/odov3/odo_test.go

```diff
@@ -53,6 +55,26 @@ func init() {
 // all stacks to be tested
 var stacks []Stack
 
+func createStack(fileName string, path string) (Stack, error) {
+	stack := Stack{id: fileName, devfilePath: path}
```

Comment:
`Stack` should probably have `version` now.
So it can be used later on line `193` where we are definite test spec name and do something like this there
```go
It(fmt.Sprintf("stack: %s starter: %s stackVersion: %s", stack.id, starterProject.Name, stack.version), func() {
```

Otherwise, there will be multiple test specs with the same name. Ginkgo should not have an issue with that, but it will be harder to see what test failed as all versions will have the same name

Reply:
Agreed, version is definitely needed here